javascript/src/main/pages/Home/Home.js

```diff
@@ -1,13 +1,15 @@
+  
 import React from "react";
 import { Jumbotron } from "react-bootstrap";
-import CourseTable from "main/components/Courses/CourseTable"
+// import CourseTable from "main/components/Courses/CourseTable"
```

Comment:
Can we remove this comment?

Reply:
I will remove this line in the next commit